packages/jest-cli/src/__tests__/generateEmptyCoverage.test.js

```diff
@@ -35,14 +38,34 @@ module.exports = {
 };`;
 
 it('generates an empty coverage object for a file without running it', () => {
+  const coverageMap = istanbulCoverage.createCoverageMap({});
+  const sourceMapStore = libSourceMaps.createSourceMapStore();
+  const rootDir = '/tmp';
+  const filepath = path.join(rootDir, './sum.js');
+
   const emptyCoverage = generateEmptyCoverage(
     src,
-    '/sum.js',
+    filepath,
     makeGlobalConfig(),
     makeProjectConfig({
       cacheDirectory: os.tmpdir(),
-      rootDir: os.tmpdir(),
+      rootDir,
+      transform: [
+        ['^.+\\.js$', path.join(__dirname, '../../../babel-jest/src/index.js')],
```

Comment:
Have to provide a `transform`. Otherwise, it will not be [transformed](https://github.com/facebook/jest/blob/master/packages/jest-runtime/src/script_transformer.js#L235).